packages/accounts-password/password_client.js

```diff
@@ -56,6 +56,27 @@ const internalLoginWithPassword = ({ selector, password, code, callback }) => {
 Meteor.loginWithPassword = (selector, password, callback) => {
   return internalLoginWithPassword({ selector, password, callback });
 };
+//this is old function around which wrapper is built
+const updatedLoginWithPassword=Meteor.loginWithPassword;
+//this is wrapper function on top of login with password to be comptable with old as well as new login style
+Meteor.loginWithPassword = function (userOrEmail, passwordOrCallback, callback) {
+ let selector, password;
+   if(typeof userOrEmail==='Object')
+   {
+     const {selector,password}=userOrEmail;
```

Comment:
why selector here? Shouldn't it be email, even username or id?

As originally expected in the issue, https://github.com/meteor/meteor/issues/13069